controllers/storagecluster/cephcluster.go

```diff
@@ -766,6 +767,35 @@ func getMonCount(sc *ocsv1.StorageCluster) int {
 	return defaults.DefaultMonCount
 }
 
+// getFullRatios returns the ratio for the given ratio name
+// If the ratio is not set in the StorageCluster CR, it returns the default value
+func getFullRatios(sc *ocsv1.StorageCluster, ratioName string) *float64 {
+	mCCSpec := sc.Spec.ManagedResources.CephCluster
+
+	switch ratioName {
+	case "nearFull":
+		if mCCSpec.NearFullRatio != nil {
+			return mCCSpec.NearFullRatio
+		}
+		return ptr.To(defaults.DefaultNearFullRatio)
+
+	case "full":
+		if mCCSpec.FullRatio != nil {
+			return mCCSpec.FullRatio
+		}
+		return ptr.To(defaults.DefaultFullRatio)
+
+	case "backfillFull":
+		if mCCSpec.BackfillFullRatio != nil {
+			return mCCSpec.BackfillFullRatio
+		}
+		return ptr.To(defaults.DefaultBackfillFullRatio)
+
+	default:
+		return nil
```

Comment:
I would suggest panic here no one should call this func with some other value. I am suggesting panic because with panic people will be able to catch it early.